config/filter.d/murmur.conf

```diff
@@ -0,0 +1,21 @@
+# Fail2Ban filter for murmur/mumble-server
+#
+
+[INCLUDES]
+
+before = common.conf
+
+
+[Definition]
+
+_daemon = murmurd
+
+failregex = Rejected connection from <HOST>:\d+: Invalid server password$
+            Rejected connection from <HOST>:\d+: Wrong certificate or password for existing user$
```

Comment:
since I am not familiar with that beast and thus not sure if any "user" input could be logged as in the line, what if we try to anchor it on both ends:
```
-
-failregex = Rejected connection from <HOST>:\d+: Invalid server password$
-            Rejected connection from <HOST>:\d+: Wrong certificate or password for existing user$
+_prefix = <W>[\n\s]*(\.\d{3})?\s+\d* => <\d*:test\(-1\)>\s+Rejected connection from <HOST>:\d+:
+failregex = ^%(_prefix)s Invalid server password$
+            ^%(_prefix)s Wrong certificate or password for existing user$
```
this patch should make it work... a bit obscure but passes the fail2ban-regex testing. `(\.\d{3})?` is there to match possibly left unmatched milliseconds, `[\n\s]` is because time gets stripped but during matching both preceding and following it parts of line merged  with '\n'